packer/common/template.dhall

```diff
@@ -3,13 +3,17 @@
 let Types = ./types.dhall
 let Functions = ./functions.dhall
 
+
 -- Global Vars
 let common_scripts =
      [
-     "scripts/vagrant.sh"
-     ,"scripts/guestAdd.sh"
-     ,"scripts/cleanbox.sh"
-     ,"scripts/end.sh"
+     "scripts/start_deb.sh",
```

Comment:
start_deb.sh and stop_deb.sh should not be in the common template but rather in a specific os base file, like the os one.

Reply:
Something like the agent target (https://github.com/Normation/rudder-tests/blob/master/packer/agent/agent.dhall).
What do you think?